cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/testdata/cert/ca-csr.json

```diff
@@ -0,0 +1,8 @@
+{
+  "CN": "fluxv2plugin-testdata-ssl-svc.default.svc.cluster.local CA",
```

Comment:
nitpick  (same in the rest of ocurrences)

```suggestion
  "CN": "helmplugin-testdata-ssl-svc.default.svc.cluster.local CA",
```

Reply:
Will change it in the following PR that is based in this branch, if you don't mind.